scripts/mvftoms.py

```diff
@@ -143,76 +147,119 @@ def main():
                      'bpcal': 'CALIBRATE_BANDPASS,CALIBRATE_FLUX',
                      'target': 'TARGET'}
 
-    usage = "%prog [options] <dataset> [<dataset2>]*"
+    def casa_style_int_list(val, argparse=False, opt_unit="m"):
```

Comment:
What is the point of `opt_unit` for antenna lists?

Reply:
This is a generic range parser. We could also possibly select only ranges of baselines to dump in the future?

Reply:
opt unit can be a list of symbols to append, e.g. 0~100[m...]?